EXAMPLES.md

```diff
@@ -260,6 +260,85 @@ There are a few things we're still working out for the web checkout experience:
 1. Handling HTTP errors gracefully. When the user loses their internet connection there is currently no way to recover.
 2. Validating the web checkout purchase with results from the API. The completion callback is invoked when we detect that the user has navigated to the `thank you page`. This is the page that is shown when a checkout is completed but we don't validate the completion of the checkout with the server yet. Due to spoofing concern I wouldn't rely on this as knowledge of a completely validated purchase _yet_.
 
+### Apple Pay Checkout - In progress
+
+After creating an instance of `Cart` and adding items to it, you can use the `CanCheckoutWithNativePay` method to 
```

Comment:
I think this is ok for now. But it would be good that later we come back to this documentation and make it so that it's literally one or two sentences describing what they'll learn. Then a code snippet that teaches what you just described. Then a longer write up about the details.